package.json

```diff
@@ -15,10 +15,11 @@
   "devDependencies": {
     "aws-sdk": "^2.636.0",
     "serverless-bundle": "^1.2.5",
-    "serverless-dotenv-plugin": "^2.1.1",
+
     "serverless-offline": "^5.3.3"
   },
   "dependencies": {
-    "mongoose": "^5.9.3"
+    "mongoose": "^5.9.3",
+    "serverless-dotenv-plugin": "^2.1.1"
```

Comment:
is it? I would have though its all done at deployment time and not at runtime (therefore still a devdep)

Reply:
"serverless-dotenv-plugin": "^2.1.1" 

I added that to dependencies is because I want to see if it will pack in the .env.prod file I set up in github action because during local deploy apparently this plugin take the .env file and inject all envs for the lambdas.

Reply:
maybe I also need to add the serverless-webpack plugin to the dependencies to compact the functions but can be implemented later, right now just want to see if this works in github actions.

Reply:
all those actions should be happening at the bundling and deployment time though, the (which devDependencies will be present for). the dependencies are only required for stuff that in used during runtime (like mongoose).

Basically if you dont import/require it, its a devDependency.

At the end of the day, given we are webpacking and serverless is taking care of the deployment, its likely that it doesnt matter and everything could be either a dependency or a devDependency and everything will still work.